Package.swift

```diff
@@ -16,17 +16,19 @@ let package = Package(
     ],
     products: [
         .library(
-            name: "SwiftCache",
-            targets: ["SwiftCache"]
+            name: "ResourceProvider",
+            targets: ["ResourceProvider"]
         )
     ],
     targets: [
         .target(
-            name: "SwiftCache"
+            name: "ResourceProvider"
         ),
         .testTarget(
-            name: "SwiftCacheTests",
-            dependencies: ["SwiftCache"]
+            name: "ResourceProviderTests",
+            dependencies: ["ResourceProvider"],
+            resources: [.process("Resources")]
         )
```

Comment:
neither of these targets are running in Swift 6 language mode. To do that, you need [flags that look like this](https://github.com/dfed/swift-async-queue/blob/c5c425e80ddc8bee18699a9007164bb38ff1ab41/Package.swift#L26-L28)